containers/jenkins-master/config/jobs/github-snappy-daily-ppa/config.xml

```diff
@@ -80,7 +80,7 @@ sudo apt build-dep -y ./
 
 rm -rf ../*.deb ../*.changes ../*.dsc ../*.tar.xz ../*.upload
 
-dch --distribution xenial -m --local +ppa$BUILD_NUMBER- Daily release to the PPA
+dch --distribution xenial -m --local +ppa$BUILD_NUMBER.$GIT_COMMIT- Daily release to the PPA
```

Comment:
Thinking about this again, maybe `$(echo $GIT_COMMIT|cut -b1-8)` ? This makes the version string a bit less unwieldy.